docs/operating-scylla/procedures/cassandra-to-scylla-migration-process.rst

```diff
@@ -259,73 +259,73 @@ Notable differences:
-|  'org.apache.cassandra.io.compress.LZ4Compressor'}            | 'org.apache.cassandra.io.compress.LZ4Compressor'}             |
-|  AND speculative_retry = '99PERCENTILE'                       | AND speculative_retry = '99.0PERCENTILE'                      |
-+---------------------------------------------------------------+---------------------------------------------------------------+
-|**AND crc_check_chance = 1.0**                                 |(Remove from Apache Cassandra 2.1 schema)                      |
-+---------------------------------------------------------------+---------------------------------------------------------------+
-|.. code-block:: cql                                            |.. code-block:: cql                                            |
-|                                                               |                                                               |
-| AND dclocal_read_repair_chance = 0.1                          | AND dclocal_read_repair_chance = 0.1                          |
-| AND default_time_to_live = 0                                  | AND default_time_to_live = 0                                  |
-| AND gc_grace_seconds = 864000                                 | AND gc_grace_seconds = 864000                                 |
-| AND max_index_interval = 2048                                 | AND max_index_interval = 2048                                 |
-| AND memtable_flush_period_in_ms = 0                           | AND memtable_flush_perios_in_ms = 0                           |
-| AND min_index_interval = 128                                  | AND min_index_interval = 128                                  |
-| AND read_repair_chance = 0.0;                                 | AND read_repair_chance = 0.0;                                 |
-+---------------------------------------------------------------+---------------------------------------------------------------+
++---------------------------------------------------------------+-----------------------------------------------------------------+
+|Apache Cassandra 3.10 (uses 3.x Schema)                        |Scylla 2.x 1.x (uses Apache Cassandra 2.1 Schema)                |
++===============================================================+=================================================================+
+|.. code-block:: cql                                            |.. code-block:: cql                                              |
+|                                                               |                                                                 |
+| CREATE KEYSPACE mykeyspace WITH replication =                 | CREATE KEYSPACE mykeyspace WITH replication =                   |
+| {'class': 'SimpleStrategy', 'replication_factor': '1'}        | {'class': 'NetworkTopologyStrategy', 'replication_factor': '1'} |
```

Comment:
The table summarizes historical differences between the 2, unless we really had NetworkTopologyStrategy back then, there's no reason to change it. OTOH RF stayed untouched, not sure if we should do anything about it either.

Reply:
OK, so it's enough for @KrzaQ to just revert this change, right?